### PR TITLE
example requesttask

### PR DESCRIPTION
**Pull request by rhundal7**
- This workflow makes a GET request to a given url

### CODE REVIEW

File: examples/example-requesttask/nodes/start.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "id": "3671f78e-9812-4da5-85d1-25c5c0551e85",
+  "name": "Start",
+  "description": "Start of Quick Request Py example Workflow",
```
Comment (kaladay) [inline examples/example-requesttask/nodes/start.json:4]:
```suggestion
  "description": "Start of Example RequestTask Workflow.",
```

File: examples/example-requesttask/triggers/startTrigger.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "id": "29f5d008-2b7c-43f4-997d-f1d1b5bee3d4",
+    "name": "Quick requesttask Test Start Trigger",
```
Comment (kaladay) [inline examples/example-requesttask/triggers/startTrigger.json:3]:
```suggestion
    "name": "Example RequestTask Start Trigger",
```

File: examples/example-requesttask/triggers/startTrigger.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+    "description": "REST Post Endpoint",
```
Comment (kaladay) [inline examples/example-requesttask/triggers/startTrigger.json:4]:
```suggestion
    "description": "REST Post Endpoint.",
```

File: examples/example-requesttask/workflow.json [hunk 4]
```diff
@@ -0,0 +1,21 @@
+
+{
+  "id": "eae461cd-9ae1-422a-8e96-60e12a8fdbf4",
+  "name": "Send GET Request Ruby",
```
Comment (kaladay) [inline examples/example-requesttask/workflow.json:4]:
```suggestion
  "name": "Example RequestTask Workflow",
```

File: examples/example-requesttask/workflow.json [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+  "description": "This workflow makes a url request and prints a response.",
```
Comment (kaladay) [inline examples/example-requesttask/workflow.json:5]:
```suggestion
  "description": "This workflow makes a URL request and prints a response.",
```

**Review by kaladay — CHANGES_REQUESTED** [review]

File: examples/README.md [hunk 6]
```diff
@@ -17,3 +17,41 @@ The `wd` variable in the `fw-cli` configuration should then look something like
...
+
+To manually execute via:
+```shell
+fw run example-requesttask
+```
+
+Trigger the workflow using an **HTTP** request such as with **Curl**:
+
+```shell
+curl --location --request POST 'http://localhost:9001/mod-workflow/events/workflow/example-requesttaskt/start' \
```
Comment (kaladay) [inline examples/README.md:52]:
```suggestion
curl --location --request POST 'http://localhost:9001/mod-workflow/events/workflow/example-requesttask/start' \
```

My previous suggestion had a typo with a spurious `t` in the URL path.

File: examples/README.md [hunk 7]
```diff
@@ -17,3 +17,41 @@ The `wd` variable in the `fw-cli` configuration should then look something like
...
+```shell
+fw config set exampleUrlPath ***
+```
+
+These variables are required when triggering the workflow:
+
+| Variable Name    | Allowed Values | Short Description |
+| ---------------- | -------------- | ----------------- |
+| exampleUrlPath   | URL            | The URL to send a GET request to. |
+| logLevel         | [INFO,DEBUG]   | Desired log level |
```
Comment (kaladay) [inline examples/README.md:35]:
```suggestion
| logLevel         | [INFO,DEBUG]   | Desired log level. |
```

File: examples/README.md [hunk 8]
```diff
@@ -17,3 +17,41 @@ The `wd` variable in the `fw-cli` configuration should then look something like
...
+```
+
+Trigger the workflow using an **HTTP** request such as with **Curl**:
+
+```shell
+curl --location --request POST 'http://localhost:9001/mod-workflow/events/workflow/example-requesttaskt/start' \
+  --header 'Content-Type: application/json' \
+  --header 'X-Okapi-Tenant: diku' \
+  --data-raw '{ "logLevel": "INFO" }'
+
```
Comment (kaladay) [inline examples/README.md:56]:
```suggestion
```

File: examples/example-requesttask/nodes/end.json [hunk 9]
```diff
@@ -0,0 +1,6 @@
+{
+  "id": "580ba4d7-5096-4f04-99bd-d12c1d091df5",
+  "name": "End",
+  "description": "End of Workflow.",
```
Comment (kaladay) [inline examples/example-requesttask/nodes/end.json:4]:
```suggestion
  "description": "End of workflow.",
```

File: examples/example-requesttask/nodes/getRequest.json [hunk 10]
```diff
@@ -0,0 +1,24 @@
+{
+  "id": "f99d0430-3028-4b40-9eb3-c9bbf8799c5a",
+  "name": "Send GET Request",
+  "description": "Send GET Request to a Given Server.",
```
Comment (kaladay) [inline examples/example-requesttask/nodes/getRequest.json:4]:
```suggestion
  "description": "Send GET request to a given server.",
```

**Review by kaladay — CHANGES_REQUESTED** [review]

File: examples/example-requesttask/nodes/rb/printResponseToScreen.rb [hunk 11]
```diff
@@ -0,0 +1,3 @@
+require 'json'
+
```
Comment (kaladay) [inline examples/example-requesttask/nodes/rb/printResponseToScreen.rb:2]: Is the `json` import needed here?
I don't see it being used.

File: examples/example-requesttask/nodes/start.json [hunk 12]
```diff
@@ -0,0 +1,8 @@
+{
+  "id": "3671f78e-9812-4da5-85d1-25c5c0551e85",
+  "name": "Start",
+  "description": "Start of Example RequestTask Workflow.",
```
Comment (kaladay) [inline examples/example-requesttask/nodes/start.json:4]:
```suggestion
  "description": "Start of Example RequestTask workflow.",
```

File: examples/example-requesttask/triggers/startTrigger.json [hunk 13]
```diff
@@ -0,0 +1,9 @@
+{
+    "id": "29f5d008-2b7c-43f4-997d-f1d1b5bee3d4",
+    "name": "Example RequestTask Start Trigger",
+    "description": "REST Post Endpoint.",
```
Comment (kaladay) [inline examples/example-requesttask/triggers/startTrigger.json:4]:
```suggestion
    "description": "REST POST end point.",
```

**Review by kaladay — CHANGES_REQUESTED** [review]

File: examples/example-requesttask/nodes/rb/printResponseToScreen.rb (hunk 11 above)
Comment (rhundal7) [reply examples/example-requesttask/nodes/rb/printResponseToScreen.rb:2]: It is not needed, I can remove it.

**Review by kaladay — APPROVED** [review]

**Pull request merged**